server_addon/houdini/client/ayon_houdini/plugins/publish/collect_render_products.py

```diff
@@ -25,7 +25,7 @@ class CollectRenderProducts(plugin.HoudiniInstancePlugin):
     """
 
     label = "Collect Render Products"
-    order = pyblish.api.CollectorOrder + 0.4
+    order = pyblish.api.CollectorOrder + 0.04
```

Comment:
Why is this needed? (I also wonder why it was +0.4 originally now)

Reply:
> why it was +0.4 originally

I don't know and  I didn't go through this plugin before as it's used for `usdrender` product type which didn't work on my side (before your PR). 

> Why is this needed?

because of `collect local render instances` expects some data the are collected by `collect render products`. 
https://github.com/ynput/ayon-core/blob/a8af05275b4d503847a8b9241018343f4a938b3e/server_addon/houdini/client/ayon_houdini/plugins/publish/collect_local_render_instances.py#L18-L19

Reply:
If it works with this collector order for the different submission types - fine with me.

Reply:
I didn't notice any difference with different render targets.